src/game.js

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import {makeShuffledDeck} from "./utils"
+
+class Game extends React.Component {
```

Comment:
Naming convention is first upper case letter for Component files, just like for the class. Since you have components outside of App.js, I recommend making a "components" folder within the "src" folder to place them!